<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoxManager::MainWindow</class>
 <widget class="QMainWindow" name="BoxManager::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>86BoxMan</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListView" name="MachineList">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>240</height>
       </size>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="autoScrollMargin">
       <number>16</number>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>358</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuManager">
    <property name="title">
     <string>Manager</string>
    </property>
    <addaction name="actionNew_Machine"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh_Machine_List"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuMachine">
    <property name="title">
     <string>Machine</string>
    </property>
    <addaction name="actionStart_Machine"/>
    <addaction name="actionConfigure_Machine"/>
    <addaction name="separator"/>
    <addaction name="actionCtrl_Alt_Delete"/>
    <addaction name="actionHard_Reset"/>
    <addaction name="separator"/>
    <addaction name="actionPress_Power_Button"/>
    <addaction name="actionHard_Power_Off"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Directory"/>
   </widget>
   <addaction name="menuManager"/>
   <addaction name="menuMachine"/>
  </widget>
  <action name="actionNew_Machine">
   <property name="text">
    <string>New Machine...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionStart_Machine">
   <property name="text">
    <string>Start Machine</string>
   </property>
  </action>
  <action name="actionConfigure_Machine">
   <property name="text">
    <string>Configure Machine</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionRefresh_Machine_List">
   <property name="text">
    <string>Refresh Machine List</string>
   </property>
  </action>
  <action name="actionOpen_Directory">
   <property name="text">
    <string>Open Directory...</string>
   </property>
  </action>
  <action name="actionCtrl_Alt_Delete">
   <property name="text">
    <string>Ctrl Alt Delete</string>
   </property>
  </action>
  <action name="actionHard_Reset">
   <property name="text">
    <string>Hard Reset</string>
   </property>
  </action>
  <action name="actionPress_Power_Button">
   <property name="text">
    <string>Press Power Button</string>
   </property>
  </action>
  <action name="actionHard_Power_Off">
   <property name="text">
    <string>Hard Power Off</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
